app/src/main/java/com/iron/espresso/domain/repo/UserRepository.kt

```diff
@@ -49,4 +49,6 @@ interface UserRepository {
     ): Single<BaseResponse<Nothing>>
 
     fun getAddressList(): Single<List<Address>>
+
+    fun verifyEmail(): Single<BaseResponse<Nothing>>
```

Comment:
```suggestion
    fun verifyEmail(): Single<'TODO'>
```
domain 레이어에 있는 클래스니 만큼 다른 레이어에있는 BaseResponse 보다는 적절히 변경해주는게 좋을 것 같습니다

Reply:
Pair<Boolean, String> 로 수정하여 result값과 messge를 받을 수 있게 했습니다
@wswon 908ea71